src/backend/aspen/api/schemas/usergroup.py

```diff
@@ -8,11 +8,14 @@
 
 
 class GroupCreationRequest(BaseRequest):
-    name: constr(min_length=3, max_length=128, strict=True)  # type: ignore
-    prefix: constr(min_length=2, max_length=20, strict=True)  # type: ignore
-    address: Optional[constr(min_length=1, max_length=128, strict=True)]  # type: ignore
-    division: Optional[constr(min_length=1, max_length=128, strict=True)]  # type: ignore
-    location: Optional[constr(min_length=1, max_length=128, strict=True)]  # type: ignore
+    name: constr(min_length=3, max_length=1000, strict=True)  # type: ignore
```

Comment:
I get address/location/etc being long, but are we sure we need to make group *name* so long? This name is displayed in the navbar for example and I don't think we've had any cases where groups pushed the 128 char limit, let alone a 200 char limit.